starter-code/iteration3/iteration3/src/app/app.component.html

```diff
@@ -0,0 +1,21 @@
+<h1>Curling Championship</h1>
+<ul>
+
+  <div class="container">
+
+    <li class="first-li" *ngFor="let oneCurling of curlingChampionship">
+      <h2>{{oneCurling.year}}</h2>
+      <h2>{{oneCurling.name}}</h2>
+      <img src="{{oneCurling.url}}">
+      <h3>Best Players</h3>
+      <!-- <p>{{oneCurling.champions}}</p> -->
+      <ul>
+        <li *ngFor="let oneChamp of curlingChampionship.champions">
```

Comment:
This is not working because `curlingChampionship` is an array. You want to iterate over `oneCurling.champions`, which is the specific championship you are currently displaying :)

` <li *ngFor="let oneChamp of oneCurling.champions"> `